frontend/components/commons/forms/questions-form/fields/multi-label/ClassifierAnnotationForFeedbackTask.component.vue

```diff
@@ -0,0 +1,267 @@
+<!--
+  - coding=utf-8
+  - Copyright 2021-present, the Recognai S.L. team.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<template>
+  <div v-if="labels.length" class="annotation-area">
+    <label-search v-if="true" :searchText="searchText" @input="onSearchLabel" />
+    <div class="feedback-interactions">
+      <classifier-annotation-button
+        v-for="label in visibleLabels"
+        :id="label.class"
+        :key="`${label.class}`"
+        v-model="selectedLabels"
+        :allow-multiple="isMultiLabel"
+        :label="label"
+        :class="[
+          'label-button',
+          predictedAs?.includes(label.class) ? 'predicted-label' : null,
+        ]"
+        :data-title="label.class"
+        :value="label.class"
+        @change="updateLabels"
+      >
+      </classifier-annotation-button>
+      <template v-if="!allowToShowAllLabels">
+        <base-button
+          v-if="visibleLabels.length < filteredLabels.length"
+          class="feedback-interactions__more secondary text"
+          @click="expandLabels()"
+          >+{{ filteredLabels.length - visibleLabels.length }}</base-button
+        >
+        <base-button
+          v-else-if="
+            visibleLabels.length > maxVisibleLabels && !allowToShowAllLabels
+          "
+          class="feedback-interactions__more secondary text"
+          @click="collapseLabels()"
+          >Show less</base-button
+        >
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import { IdState } from "vue-virtual-scroller";
+import ClassifierAnnotationButton from "@/components/text-classifier/results/classifier-annotation/ClassifierAnnotationButton.vue";
+
+export default {
+  components: {
+    ClassifierAnnotationButton,
+  },
+  mixins: [
+    IdState({
+      // You can customize this
+      idProp: (vm) => {
+        return `${vm.prefix}-${vm.record.id}`;
+      },
+    }),
+  ],
+  props: {
+    maxVisibleLabels: {
+      type: Number,
+      default: () => 0,
+    },
+    record: {
+      type: Object,
+      required: true,
+    },
+    datasetName: {
+      type: String,
+      required: true,
+    },
+    isMultiLabel: {
```

Comment:
always is true, so we can remove it

Reply:
The props is not always true, because I am thinking to reuse the same copy component for `single` and `multilabel`. But this may change in the future.